crates/workspace-hack/Cargo.toml

```
@@ -0,0 +1,561 @@
+# This file is generated by `cargo hakari`.
+# To regenerate, run:
+#     cargo hakari generate
+
+[package]
+name = "workspace-hack"
```

Could we make the name/directory snake_cased like the rest of our crates?

```suggestion
name = "workspace_hack"
```

---

I kind of wanted to avoid that since we can just use the existing published one this way https://crates.io/crates/workspace-hack. I'll try it and see if path normalization makes it work though.

---

And/or could we move it into `tooling/` instead of `crates/`?

Since it is a bit different from the rest of our crates.

---

Yeah I get:
```
Error:
   0: building package graph failed
   1: `cargo metadata` execution failed
   2: `cargo metadata` exited with an error:     Updating crates.io index
      error: no matching package found
      searched package name: `workspace_hack`
      perhaps you meant:      workspace-hack
      location searched: crates.io index
```

Do we wanna publish a `zed_workspace_hack` crate on crates.io so we can have our naming convention?

---

Does publishing the crate bring any additional maintenance overhead with it? Or is it mostly a one-time thing?

I'm okay with the current name of `workspace-hack` if we're able to put it in `tooling/workspace-hack` instead of in `crates/workspace-hack.

---

It should be one-time but I'd prefer to avoid publishing an extra crate so I'll go ahead and stick it in tooling.